stage1/units/units/supervisor-ready.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=rkt supervisor-ready signaling
+DefaultDependencies=no
+Requires=default.target
+Before=halt.target
+Conflicts=halt.target
+
+[Service]
+Type=oneshot
+Restart=no
+ExecStart=/bin/bash -c "systemd-tmpfiles --create  <(echo L+ /rkt/supervisor-status - - - - ready)"
+ExecStop=/bin/bash -c "systemd-tmpfiles --remove  <(echo R /rkt/supervisor-status)"
```

Comment:
Thanks for the heads-up. I also saw some regressions on the stage1-host flavor, so I'll put up a cumulative PR to fix all of those. I think it is safe to enable systemd-tmpfiles as you did, but I have to double-check if it also installs additional assets.